PetbookAngular/src/app/petbook/petbook.component.html

```diff
@@ -84,7 +84,24 @@ <h3>{{p.name}}</h3>
           </ul>
         </ng-container>
       </mat-card-content>
-    </mat-card>    
+    </mat-card>
+  </div>
```

Comment:
Delete these lines and then you will have friends and suggested friends stacked in a single column